### PR TITLE
Add PyMOL Open Source

### PR DESCRIPTION
**Pull request by padix-key**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by padix-key** [comment]
I think this recipe is now ready to be merged.

File: recipes/pymol-open-source/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  build:
+    - python
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - mesalib                        # [linux]
+    - libglu                         # [linux]
+    - libpng
+    - freetype
+    - pyqt
+    - glm
```
Comment (chrisburr) [inline recipes/pymol-open-source/meta.yaml:29]: Most of these aren't needed in build:
```suggestion
    - {{ compiler('c') }}
    - {{ compiler('cxx') }}
```

File: recipes/pymol-open-source/LICENSE.txt [hunk 2]
```diff
@@ -0,0 +1,126 @@
+ Open-Source PyMOL Copyright Notice
```
Comment (chrisburr) [inline recipes/pymol-open-source/LICENSE.txt:1]: No need for this file as it's already in the source tarball

File: recipes/pymol-open-source/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  files:
+    - 1aki.cif
+    - test.py
+  commands:
+    - "python test.py"
+
+about:
+  home: https://pymol.org/
+  license: LicenseRef-PyMOL
+  license_file: LICENSE.txt
```
Comment (chrisburr) [inline recipes/pymol-open-source/meta.yaml:63]:
```suggestion
  license_file: LICENSE
```

File: recipes/pymol-open-source/1aki.cif [hunk 4]
```diff
@@ -0,0 +1,2192 @@
+data_1AKI
```
Comment (chrisburr) [inline recipes/pymol-open-source/1aki.cif:1]: It is possible to download this file from somewhere or take it from the source tarball?

(Mostly concerned with adding a relatively large file into the staged-recipes git history)

File: recipes/pymol-open-source/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,74 @@
+{% set version = "2.4.0" %}
+{% set number = "0" %}
+{% set sha256 = "5ede4ce2e8f53713c5ee64f5905b2d29bf01e4391da7e536ce8909d6b9116581" %}
```
Comment (chrisburr) [inline recipes/pymol-open-source/meta.yaml:3]:
```suggestion
```

File: recipes/pymol-open-source/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+
+package:
+  name: pymol-open-source
+  version: {{ version }}
+
+source:
+  url: https://github.com/schrodinger/pymol-open-source/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```
Comment (chrisburr) [inline recipes/pymol-open-source/meta.yaml:11]:
```suggestion
  sha256: 5ede4ce2e8f53713c5ee64f5905b2d29bf01e4391da7e536ce8909d6b9116581
```

File: recipes/pymol-open-source/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+
+build:
+  number: {{ number }}
+  script: "{{ PYTHON }} setup.py install --use-msgpackc=no --no-libxml --no-vmd-plugins"
+  skip: True  # [py < 37]
+  skip: True  # [osx]
```
Comment (chrisburr) [inline recipes/pymol-open-source/meta.yaml:17]: How come macOS is being skipped? (it's unusual for Linux and Windows to work but not macOS)

Comment (padix-key) [reply recipes/pymol-open-source/meta.yaml:17]: On MacOS I encountered a quite confusing compiler error, which I was notable to resolve. Therefore, I excluded MacOS for now.

File: recipes/pymol-open-source/1aki.cif (hunk 4 above)
Comment (padix-key) [reply recipes/pymol-open-source/1aki.cif]: The file is now downloaded in the test itself.

File: recipes/pymol-open-source/meta.yaml (hunk 3 above)
Comment (padix-key) [reply recipes/pymol-open-source/meta.yaml:63]: The problem are the linked libraries, whose license needs to be referenced too, as far as I know.

File: recipes/pymol-open-source/meta.yaml (hunk 1 above)
Comment (padix-key) [reply recipes/pymol-open-source/meta.yaml]: You are right, I tested it without most of these packages and it still works fine. Therefore I removed them. However, Python is required in ``{{ PYTHON }} setup.py install``, am I right?

Comment (chrisburr) [reply recipes/pymol-open-source/meta.yaml]: Python should be in `host`

Comment (padix-key) [reply recipes/pymol-open-source/meta.yaml]: You are right: I removed Python from the build requirements and it still works. Thank you for your help!

**Pull request merged**